dashboard/app/views/teacher_dashboard/parent_letter.html.haml

```diff
@@ -12,9 +12,9 @@
   -# Here where we pass data down to the particular script we'd like to run
   :ruby
     teacher_dashboard_data = {}
-    teacher_dashboard_data[:studioUrlPrefix] = CDO.studio_url('', CDO.default_scheme)
```

Comment:
I thought about this a little more and I think we do need the studioUrlPrefix because we want the full text of the link to show in the descriptive text (eg. studio.code.org/join). If we use the urlHelpers, we'll link to the correct places but the text will only have the relative part of the url (/join).  I think it's fine if this is a follow-up/optimization.

Reply:
I thought that too, but when I went to read `urlHelpers` I think that it's by-design that they give you a relative url if you try to format a studio url from a studio page.  (Because `window.pegasus` shouldn't be present when we're on a studio page, as we are in this case.)

https://github.com/code-dot-org/code-dot-org/blob/9af3327fb5fdc0d46b112368bdad97265c9912cc/apps/src/lib/util/urlHelpers.js#L28-L33

Instead I've hard-coded the visible urls to production; but I think you're right, and we may still be using relative urls as hrefs, which may not work properly when we print to PDF!  Maybe I should put this back _and_ add `urlHelpers` methods that guarantee absolute urls.